### PR TITLE
added discovered hosts widget for dashboard

### PR DESCRIPTION
**Pull request by vijay8451**
- Added Discovered Host widget .

### CODE REVIEW

File: airgun/views/dashboard.py [hunk 1]
```diff
@@ -99,6 +100,11 @@ def search(self, query):
         self.searchbox.search(query)
         return self.read()
 
+    @View.nested
+    class DiscoveredHosts(View):
+        ROOT = ".//li[@data-name='Discovered Hosts']"
+        hosts = Table('.//table')
+
```
Comment (omkarkhatavkar) [inline airgun/views/dashboard.py]: @vijay8451 thanks for adding airgun PR :+1: 

Two Queries : 
1. Can you please also add the widget header as well?  It is a link containing the number of hosts ( case for assertion). 
2. There is also navigation from widget to details page, Do you think we need to add that flow as user will be doing that mostly ? 

download

Comment (vijay8451) [reply airgun/views/dashboard.py]: @omkarkhatavkar thanks ..  
for queries , 1) Added widget in last commit 2) not looks that much usable one.

**Comment by vijay8451** [comment]
> Strong + for first point in Omkar comment.
> ACK pending comment and check what PR is going to be first merged: mine or yours to resolve `Table` import conflict

Added the new widget for first point , And about `Table` so shall resolve if conflict occur

**Comment by oshtaier** [comment]
sorry, not sure why we do not have conflict with import:
https://github.com/SatelliteQE/airgun/blob/master/airgun/views/dashboard.py#L2
Maybe it is necessary to do rebase
Other than that PR is ready to be merged

**Comment by vijay8451** [comment]
> sorry, not sure why we do not have conflict with import:
> https://github.com/SatelliteQE/airgun/blob/master/airgun/views/dashboard.py#L2
> Maybe it is necessary to do rebase
> Other than that PR is ready to be merged

@oshtaier not sure why this is not showing and ideally that should occur.
I have fixed that now and good to merge it.

**Pull request merged**